frontend/www/js/omegaup/teamsgroup/edit.ts

```diff
@@ -45,6 +66,202 @@ OmegaUp.on('ready', () => {
+              },
+            })
+              .then(() => {
+                ui.success(T.teamsGroupEditTeamsUpdated);
+                this.refreshTeamsList();
+              })
+              .catch(ui.apiError);
+          },
+          'change-password-identity-team': ({
+            username,
+            newPassword,
+            newPasswordRepeat,
+          }: {
+            username: string;
+            newPassword: string;
+            newPasswordRepeat: string;
+          }) => {
+            if (newPassword !== newPasswordRepeat) {
+              ui.error(T.userPasswordMustBeSame);
+              return;
+            }
+
+            api.Identity.changePassword({
+              group_alias: payload.teamGroup.alias,
+              password: newPassword,
+              username: username,
+            })
+              .then(() => {
+                this.refreshTeamsList();
+                ui.success(T.teamsGroupEditTeamsPasswordUpdated);
+              })
+              .catch(ui.apiError);
+          },
+          remove: (username: string) => {
+            api.Group.removeUser({
+              group_alias: payload.teamGroup.alias,
+              usernameOrEmail: username,
+            })
+              .then(() => {
+                this.refreshTeamsList();
+                ui.success(T.teamsGroupEditTeamsRemoved);
+              })
+              .catch(ui.apiError);
+          },
+          'update-search-result-users': (query: string) => {
+            api.User.list({ query })
+              .then((data) => {
+                // Users previously invited to any team in the current teams
+                // group can not be added to another, so they should not be
+                // shown in the dropdown
+                // TODO: Define how to ignore users previosly mentioned
+                // const addedUsers = new Set(
+                //   this.users.map((user) => user.username),
+                // );
+
+                this.searchResultUsers = data
+                  //.filter((user) => !addedUsers.has(user.label))
+                  .map((user) => ({
+                    key: user.label,
+                    value: `${ui.escape(user.label)} (<strong>${ui.escape(
+                      user.value,
+                    )}</strong>)`,
+                  }));
+              })
+              .catch(ui.apiError);
+          },
+          'bulk-identities': ({
+            identities,
+            identitiesTeams,
+          }: {
+            identities: types.Identity[];
+            identitiesTeams: { [team: string]: string[] };
+          }) => {
+            api.Identity.bulkCreateForTeams({
+              team_identities: JSON.stringify(
+                identities.map((identity) => {
+                  return {
+                    ...identity,
+                    ...{
+                      usernames: identitiesTeams[identity.username].join(';'),
+                    },
+                  };
+                }),
```

Comment:
```suggestion
              identities.map((identity) => ({
                ...identity,
                ...{
                  usernames: identitiesTeams[identity.username].join(';'),
                },
              })),
```

`(...) => { return { ... }; }` se puede reemplazar con `(...) => ({ ... })`! (esos paréntesis extra son para que el parser de JavaScript pueda distinguir entre un bloque de código y un objeto).

Reply:
Listo.